features/bootstrap/FeatureContext.php

```diff
@@ -93,7 +93,6 @@ public function createPayloadOptions(TableNode $tableNode): void
             isset($data['immutable']) ? (bool)$data['immutable'] : null,
             isset($data['forceAccount']) ? (bool)$data['forceAccount'] : null,
             $data['returnUrl'] ? new ReturnUrl(null, $data['returnUrl']) : null,
-            $data['signers'] ? json_decode($data['signers'], true) : null,
```

Comment:
No idea. Shouldn't be, still relevant.

Reply:
`Signers` was added in a previous commit by me in this PR because I thought it was missing. `Signers` is not implemented in the scenarios and broke the GH build, so I removed it because that's not part of the pathfinding implementation.

Reply:
Oh interesting, that should work. I think the Xumm mock API is probably outdated. I'll fix that on my end.